setup.py

```diff
@@ -123,7 +123,6 @@ def run_tests(self):
     classifiers=[
         "Programming Language :: Python",
         "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 3.4",
```

Comment:
Should we add `python_requires`?

Reply:
I don't think it can hurt. I don't know if `pip` implements it but it's a good idea anyhow.